### PR TITLE
v0.1.4 bump three-dev to fix android

### PR DESCRIPTION
**Pull request by ngokevin**
r? @cvan 

Want to push out a patch release until 0.2.0.

### CODE REVIEW

**Comment by cvan** [comment]
thx, will take a look

**Comment by ngokevin** [comment]
@cvan want to help shepherd this? would be good practice for more regular releases.

- merge this, publish aframe-core
- bump aframe master to point to new aframe-core, publish aframe
- change aframe site to say support Android
- tweet it

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aframe-core",
-  "version": "0.1.3",
+  "version": "0.1.4",
```
Comment (cvan) [inline package.json]: I know you know, but just a reminder: going to have to bump [`aframe-core` in `aframe`'s `package.json`](https://github.com/aframevr/aframe/blob/master/package.json#L35)

**Comment by cvan** [comment]
I'm missing some context here. can you point me to the issue this fixes? tried several examples that use textures and videos. and before and after, I see no difference. what should I be looking for?

**Comment by ngokevin** [comment]
https://github.com/aframevr/aframe/issues/956
https://github.com/aframevr/aframe/issues/599
https://github.com/aframevr/aframe/issues/307
https://github.com/aframevr/aframe/issues/314
https://twitter.com/rvdleun/status/699502807464939520

**Comment by ngokevin** [comment]
It fixes Android issues with phones like 1+1 or MotoX.

**Comment by cvan** [comment]
gotcha, thanks - can you put one of the issues in the commit message?

it turns out I have 12 Android devices, but no OnePlus or MotoX

**Comment by cvan** [comment]
# r+

**Comment by ngokevin** [comment]
Sorry, forgot the commit message part.

**Comment by sirkitree** [comment]
will there be a release for this? I don't see 0.1.3 or 0.1.4

**Comment by ngokevin** [comment]
It's 0.1.2

**Comment by ngokevin** [comment]
There's 0.1.5 on NPM, the dists are bumped on this repo, and this has been upstreamed to aframe 0.1.2

**Pull request merged**